eZ/Publish/Core/settings/repository/inner.yml

```
@@ -190,6 +190,8 @@ services:
             $contentLanguageHandler: '@ezpublish.spi.persistence.language_handler'
             $fieldTypeRegistry: '@eZ\Publish\Core\FieldType\FieldTypeRegistry'
             $thumbnailStrategy: '@eZ\Publish\Core\Repository\Strategy\ContentThumbnail\ThumbnailChainStrategy'
+        calls:
+            - [setLogger, ['@?logger']]
```

Add `monolog.logger` tag, and maybe add a channel for it.

```suggestion
        calls:
            - [setLogger, ['@?logger']]
        tags:
            - { name: 'monolog.logger', channel: 'ibexa.core' }
        
```